src/DeVinci_frontend/components/Sidebar.svelte

```diff
@@ -12,6 +13,9 @@
 <div class="sidebar-header flex flex-col items-center justify-between py-4 h-lvh">
     <SidebarHeader />
     {#if !userHasDownloadedAtLeastOneModel}
-      <SidebarFooter />
+        <SidebarInfo />
+    {/if}
+    {#if userHasDownloadedAtLeastOneModel}
```

Comment:
this can be refactored into
{#if userHasDownloadedAtLeastOneModel}
...
{:else}
   ...
  {/if}